model/AI/AI.md

```diff
@@ -15,3 +15,14 @@ The AI profile namespace defines concepts related to AI application and model ar
 - id: https://rdf.spdx.org/v3/AI
 - name: AI
 
+## Profile conformance 
+
+For an element collection to be conformant with this profile,
+the following has to hold:
+
+1. for every `/AI/AIPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `concludedLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
+2. for every `/AI/AIPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `declaredLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
```

Comment:
Same comment as for `Dataset` conformance.

Reply:
I will discuss this with the working group. Thanks!

Reply:
keeping declaredLicense for now.